### DISCUSSION TITLE
Display .md path in search results

### DISCUSSION THREAD

**Discussion by AndyDeNike** [squidfunk/mkdocs-material#4210] [Q&A]
Hey squidfunk and community, 

This is a reference to https://github.com/squidfunk/mkdocs-material/issues/389, where I made the comment:
_Hello, I am referencing this legacy issue and notice that your links in above comment seem to be out of date (it seems you updated the .jsx files to .ts?). Would you please be able to provide updated insight on how to display full article path in search results._

_Any and all information would be extremely appreciated and your work on this app has been top tier. Thanks for everything._

I wanted to add this image for reference as well, thanks for your time.  

![IMG_1264 JPG](https://user-images.githubusercontent.com/15356537/182974292-8e708683-4889-4793-8edd-63eea18224ca.jpeg)

**Comment by squidfunk** [comment]
Please see my comments in https://github.com/squidfunk/mkdocs-material/issues/389.

__Related__:

- #3787
